src/main/kotlin/compiler/ast/N_EXPR.kt

```diff
@@ -59,7 +59,7 @@ class N_STRING_SUB(val parts: List<N_EXPR>): N_EXPR() {
         when (parts.size) {
             0 -> return
             else -> {
-                if (!parts[0].isEmptyString()) parts[0].code(coder)
+                parts[0].code(coder)
```

Comment:
BUGFIX : I over-optimized string sub coding and created a stack inconsistency when a subbed string starts with a sub.